apps/condo/bin/sbbol/credentials.js

```diff
@@ -51,6 +55,12 @@ const workerJob = async () => {
         await changeClientSecret({ clientId, currentClientSecret, newClientSecret })
     }
 
+    if (command === COMMAND.GET) {
+        const sbbolSecretStorage = getSbbolSecretStorage()
+        const values = await sbbolSecretStorage.getRawKeyValues()
+        console.log('SbbolSecretStorage values: ', JSON.stringify(values, null, 2))
```

Comment:
console.log ?

Reply:
If that logs are important you can change this to `logger.info`

Reply:
This util will be invoked manually from console.